google-cloud-monitoring-dashboard/src/test/java/com/google/cloud/monitoring/dashboard/v1/it/ITSystemTest.java

```diff
@@ -0,0 +1,120 @@
+  private static String etag;
+
+  private static final String PROJECT_ID = ServiceOptions.getDefaultProjectId();
+  private static final String PARENT = "projects/" + PROJECT_ID;
+  private static final String ID = UUID.randomUUID().toString().substring(0, 8);
+  private static final String DASHBOARD_DISPLAY_NAME = "test-dashboard-" + ID;
+  private static final GridLayout GRID_LAYOUT = GridLayout.newBuilder().setColumns(5).build();
+
+  @BeforeClass
+  public static void setUp() throws Exception {
+    client = DashboardsServiceClient.create();
+    Dashboard dashboard =
+        Dashboard.newBuilder()
+            .setGridLayout(GRID_LAYOUT)
+            .setDisplayName(DASHBOARD_DISPLAY_NAME)
+            .build();
+    CreateDashboardRequest request =
+        CreateDashboardRequest.newBuilder().setParent(PARENT).setDashboard(dashboard).build();
+    Dashboard response = client.createDashboard(request);
+    dashboardName = response.getName();
+    etag = response.getEtag();
+  }
+
+  @AfterClass
+  public static void tearDown() {
+    DeleteDashboardRequest request =
+        DeleteDashboardRequest.newBuilder().setName(dashboardName).build();
+    client.deleteDashboard(request);
+    client.close();
+  }
+
+  @Test
+  public void getDashboardTest() {
+    GetDashboardRequest request = GetDashboardRequest.newBuilder().setName(dashboardName).build();
+    Dashboard dashboard = client.getDashboard(request);
+    assertEquals(dashboardName, dashboard.getName());
+    assertEquals(DASHBOARD_DISPLAY_NAME, dashboard.getDisplayName());
+    assertEquals(GRID_LAYOUT, dashboard.getGridLayout());
+  }
+
+  @Test
+  public void updateDashboardTest() {
+    Dashboard actualDashboard =
+        Dashboard.newBuilder()
+            .setName(dashboardName)
+            .setGridLayout(GRID_LAYOUT)
+            .setDisplayName(DASHBOARD_DISPLAY_NAME)
+            .setEtag(etag)
+            .build();
+    UpdateDashboardRequest request =
+        UpdateDashboardRequest.newBuilder().setDashboard(actualDashboard).build();
+    Dashboard dashboard = client.updateDashboard(request);
+    assertEquals(dashboardName, dashboard.getName());
+    assertEquals(etag, dashboard.getEtag());
+    assertEquals(DASHBOARD_DISPLAY_NAME, dashboard.getDisplayName());
+    assertEquals(GRID_LAYOUT, dashboard.getGridLayout());
+  }
+
+  @Test
+  public void listDashboardsTest() {
+    ListDashboardsRequest request = ListDashboardsRequest.newBuilder().setParent(PARENT).build();
+    for (Dashboard actualDashboard : client.listDashboards(request).iterateAll()) {
+      if (dashboardName.equals(actualDashboard.getDisplayName())) {
+        assertEquals(dashboardName, actualDashboard.getName());
+        assertEquals(DASHBOARD_DISPLAY_NAME, actualDashboard.getDisplayName());
+        assertEquals(GRID_LAYOUT, actualDashboard.getGridLayout());
+      }
+    }
+  }
+
+  @Test
+  public void isShutdownTest() {
+    boolean isShutdown = client.isShutdown();
+    assertFalse(isShutdown);
```

Comment:
I think merging both lines should be fine
`assertFalse(client.isShutdown());`

Reply:
correct but i am moving ahead with two line because its more readable.